python-sdk/nuscenes/map_expansion/arcline_path_utils.py

```diff
@@ -21,7 +25,7 @@ def principal_value(angle_in_radians: float) -> float:
     return scaled_angle
 
 
-def compute_segment_sign(arcline_path_3: Dict[str, Any]) -> Tuple[int, int, int]:
+def compute_segment_sign(arcline_path_3: ArcLinePath) -> Tuple[int, int, int]:
```

Comment:
Let's be consistent with the name. Maybe just `ArcLinePath`? We can remove the 3 to clarify the name, IMO.

Reply:
Good idea!